src/config/mod.rs

```diff
@@ -340,6 +340,7 @@ mod tests {
 
     fn ipc_plugin() -> IpcPlugin {
         let cfg = IpcPluginConfig {
+            ipc_to_suricata_channel_size: 1_000,
```

Comment:
lets default to 1

Reply:
That's actually in test code. Will default to 1 though.